lineapy/api/api.py

```diff
@@ -503,7 +505,9 @@ def get_function(
         input_parameters=input_parameters,
         reuse_pre_computed_artifacts=reuse_pre_computed_artifacts,
     )
-    return art_collection.get_module().run_all_sessions
+    writer = BasePipelineWriter(art_collection)
+    module = load_as_module(writer)
+    return module.run_all_sessions
```

Comment:
So you agree function is a flavor of pipeline right? :)

Reply:
Ideally we don't have to go through the writer to be able to get the Pipeline as a runnable module. 

I can't figure a quick way to implement this. So for now yes, I'm using BasePipelineWriter as a "function flavor" for Pipeline.